build/build.ps1

```diff
@@ -210,9 +218,18 @@ function Build {
 
   echo "Repo toolset used from: $RepoToolsetBuildProj"
 
-  $solution = Join-Path $RepoRoot "MSBuild.sln"
+  $createTlb = $true
+  if ($DotNetBuildFromSource)
+  {
+    $solution = Join-Path $RepoRoot "MSBuild.SourceBuild.sln"
+    $createTlb = $false
+  }
+  else
+  {
+    $solution = Join-Path $RepoRoot "MSBuild.sln"
+  }
 
-  $commonMSBuildArgs = "/m", "/clp:Summary", "/v:$verbosity", "/p:Configuration=$configuration", "/p:Projects=$solution", "/p:CIBuild=$ci", "/p:RepoRoot=$reporoot"
+  $commonMSBuildArgs = "/m", "/clp:Summary", "/v:$verbosity", "/p:Configuration=$configuration", "/p:Projects=$solution", "/p:CIBuild=$ci", "/p:RepoRoot=$reporoot", "/p:DisableNerdbankVersioning=$DotNetBuildFromSource", "/p:CreateTlb=$createTlb"
```

Comment:
The NerdbankVersioning restriction comes from Redhat. Is this still valid for Windows?

Reply:
I've deleted this as we're already passing this msbuild parameter through source-build: 

https://github.com/dotnet/source-build/blob/dev/release/2.1/repos/msbuild.proj#L9